x-pack/plugin/ilm/qa/multi-cluster/build.gradle

```diff
@@ -8,13 +9,20 @@ dependencies {
     testCompile project(':x-pack:plugin:ilm')
 }
 
-task leaderClusterTest(type: RestIntegTestTask) {
+File repoDir = file("$buildDir/testclusters/repo")
+
+task 'leader-cluster'(type: RestIntegTestTask) {
```

Comment:
why rename this to `leader-cluster` instead of `leaderCluster` so that it requires quoting everywhere? Same with `follower-cluster`. It seems really awkward to have all the quoting.

Reply:
With test-clusters the name of the cluster has to be set up-front when it's defined and can't be changed. The  current convention is  that the cluster will have the same name as the task, we had this cluster name before so I just kept it. We could use any other name, we would just have to update the tests to use these names. CCR tests do currently use the quoted names as it's done here.